Index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto&family=Roboto:400;700&subset=latin&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="stylesheet.css">
+
+    
+  </head>
+  <body>
+    <div class="outer-maincont">
+        <header class="app-bar">
+            <nav class="lower-bar">
+                <ul><li><a href="#" class="logoo">Somarib Emmanuel</a></li>
+                    <li><a href="#" id="mbm-icon"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li>
+                </ul>
+            </nav>
+        </header>
```

Comment:
[OPTIONAL] Please use flexbox to align the items for your header, as we need you to be a flex ninja, read about [flex here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.